### ISSUE TITLE
Javascript / CSharp Inter-operation

### ISSUE THREAD

**Issue by jingyiliu** [CefNet/CefNet#6]
Hi, how to call a js function from cs code, and how to call cs from js?

**Comment by vmas** [comment]
CefNet is a .NET wrapper around the Chromium Embedded Framework (CEF). It provides similar APIs that CEF provides. See https://bitbucket.org/chromiumembedded/cef/wiki/JavaScriptIntegration.md

**Comment by vmas** [comment]
CefNet also provides the ScriptableObject, but it only works with simple types and does not yet allow CS calls from JS. See [example](https://github.com/CefNet/CefNet/blob/343cade932b0b8059589ba9a29134ada5dcc2d8f/WinFormsCoreApp/ScriptableObjectTests.cs#L36). Do not use any dynamic operations on the UI thread or any CEF thread.

**Comment by jingyiliu** [comment]
Hi, vmas, thanks for the quickly response. I've checked the link you provided above.
It seems that the interop ability of CefNet is somehow basic right now.

I've tested CefSharp and CefGlue personally, the CefSharp is good, makes cs/js interop really simple, but as it use c++/cli, it's not cross platform.

The CefGlue, on the other hand, is totally cross platform, but inconvenient for interop.

And there are another CefGlue fork, called Chromely, This one provide some enhancement for CefGlue, but it's targeting native GUI, HTML5, JavaScript, CSS, saddly
